src/App.stories.tsx

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+/** @jsxRuntime classic */
```

Comment:
Can we use [`@babel/plugin-transform-react-jsx`](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#usage) to avoid this by setting the run time at a higher level?